src/client.rs

```diff
@@ -231,12 +231,11 @@ impl Client {
             for (info, node_url) in nodes.iter() {
                 if let Ok(mut client_network_info) = network_info.write() {
                     client_network_info.network_id = hash_network(&info.protocol.network_name).ok();
-                    // todo update protocol version
                     client_network_info.min_pow_score = Some(info.protocol.min_pow_score);
                     client_network_info.bech32_hrp = Some(info.protocol.bech32_hrp.clone());
                     client_network_info.rent_structure = Some(info.protocol.rent_structure.clone());
                     if !client_network_info.local_pow {
-                        if info.features.contains(&"Pow".to_string()) {
+                        if info.features.contains(&"PoW".to_string()) {
```

Comment:
So are we doing this or not ?

Reply:
confused too

Reply:
`Pow` is currently wrong, I asked if we should change it to that in the node and so far only one person said no